painters/model_def_standard.py

```diff
@@ -0,0 +1,134 @@
+import os
+
+import keras
+import numpy as np
+import tensorflow as tf
+from keras import backend as K
+from keras.datasets.cifar import load_batch
+from keras.layers import (Activation, Conv2D, Dense, Dropout, Flatten,
+                          MaxPooling2D)
```

Comment:
`keras.layers` imports are unused. Remove these and any other imports that aren't used in this file.